apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -96,19 +96,30 @@ const loadListByChunks = async ({
     list,
     where = {},
     sortBy = ['createdAt_ASC'],
-    chunkSize = 100,
```

Comment:
O_O you don't need to change the public API of existing function.

Reply:
I didn't change API, I added some checks